hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/metrics2/impl/TestStatsDMetrics.java

```diff
@@ -31,13 +31,13 @@
 import java.util.List;
 import java.util.Set;
 
+import org.apache.commons.lang3.reflect.FieldUtils;
```

Comment:
We should use getter and setter instead of relying on the FieldUtils.

Reply:
`org.apache.commons.lang3.reflect.FieldUtils` has been removed, and TestStatsDMetrics.java has also been moved to the package org.apache.hadoop.metrics2.sink.